doc/connectivity/bluetooth/api/mesh/sar_cfg.rst

```diff
@@ -0,0 +1,243 @@
+reassembly mechanism is used by the lower transport layer.
+
+The lower transport layer takes Upper Transport PDUs from the upper transport layer and sends them to the lower transport layer on a peer device.
+If the Upper Transport PDU fits, it is sent in a single Lower Transport PDU.
+For longer packets, which do not fit into a single Lower Transport PDU, the lower transport layer performs segmentation, splitting the Upper Transport
+PDU into multiple segments.
+
+The receiving lower transport layer on the other device reassembles the segments into a single Upper Transport PDU before passing it
+up the stack. Delivery of a segmented message is acknowledged by the lower transport layer of the receiving node.
+
+Delivery of an unsegmented message is not acknowledged.
+But, an Upper Transport PDU that fits into one Lower Transport PDU can also be sent as a single-segment segmented message when acknowledgment
+by the lower transport layer is required.
+Set the ``send rel`` flag (see :c:struct:`bt_mesh_msg_ctx`) to use the reliable message transmission and acknowledge single-segment segmented messages.
+
+The transport layer is able to transport up to 32 segments with its SAR mechanism, with a maximum message (PDU) size of 384 octets.
+To configure number of segments and the message size for the Bluetooth mesh stack, use the following Kconfig options:
+
+* :kconfig:option:`CONFIG_BT_MESH_SEG_BUFS` to set the number of segment buffers.
+* :kconfig:option:`CONFIG_BT_MESH_RX_SEG_MAX` to set the maximum number of segments in an incoming message.
+* :kconfig:option:`CONFIG_BT_MESH_TX_SEG_MAX` to set the maximum number of segments in an outcoming message.
+
+Never more than one segmented message is transmitted at the time, but the Bluetooth mesh stack allows you to define buffers that will queue a defined
+number of ingoing or outgoing segmented messages.
+Use Kconfig options :kconfig:option:`CONFIG_BT_MESH_TX_SEG_MSG_COUNT` and :kconfig:option:`CONFIG_BT_MESH_RX_SEG_MSG_COUNT` to set the maximum number
+of queued outgoing and incoming segmented messages respectively.
+
+Incoming and outgoing segmented messages share the same segment buffer, which means that they allocate their segments from the same pool.
+Both incoming and outgoing messages allocate segments at the start of the transaction, but release them at the different time.
+While the outgoing segmented message releases its segments one by one as soon as they are acknowledged by the receiver, the incoming message will not
+release the segments until the message is fully received.
+Keep this in mind when defining the size of the buffers.
+
+SAR does not impose extra overhead on the access layer payload per segment.
+
+Segmentation and reassembly (SAR) Configuration models are foundation models defined by the
+`Bluetooth Mesh Protocol Specification version 1.1 <https://www.bluetooth.com/specifications/mesh-specifications/>`_.
```

Comment:
This is the first and only place the spec is referenced on this page.
I am not sure what page you mean, maybe mesh.rst?
I also see we are only mentioning v1.0.1 there, no 1.1.

Reply:
> This is the first and only place the spec is referenced on this page.
I am not sure what page you mean, maybe mesh.rst?

Yes, I meant mesh.rst. Otherwise, we need to add the reference to every model's doc page which is too much I think. 

> I also see we are only mentioning v1.0.1 there, no 1.1.

This should be fixed (not necessarily in your PR, but up to you).